bundle/edu.gemini.pit/src/main/resources/edu/gemini/pit/ui/editor/institutions.xml

```diff
@@ -3421,6 +3428,23 @@
             <email>mbailes@swin.edu.au</email>
         </contact>
     </site>
+    <site>
+        <institution>Texas A&amp;M University</institution>
+        <address>Department of Physics</address>
+        <address>College Staton</address>
+        <address>TX</address>
+        <address>77843</address>
+        <country>USA</country>
+    </site>
+    <site>
+        <institution>University of Texas at Austin</institution>
+        <address>Department of Astronomy</address>
+        <address>1 University Station C-1400</address>
+        <address>Austin</address>
+        <address>TX</address>
+        <address>77843</address>
```

Comment:
According to http://jira.gemini.edu:8080/browse/REL-3270, the ZIP code should be 78712.
I think you accidentally used the one for `Texas A&M University` twice.

Reply:
Fixed! Thanks for double checking.